charts/alfresco-search-enterprise/Chart.yaml

```diff
@@ -3,8 +3,8 @@ apiVersion: v2
 name: alfresco-search-enterprise
 description: A Helm chart for deploying Alfresco Elasticsearch connector
 type: application
-version: 4.4.3
-appVersion: 5.0.0
+version: 4.5.0
```

Comment:
same thing here, from https://hyland.atlassian.net/wiki/spaces/TECH/pages/1293317631/Charts+release+versioning+scheme#Bugfixes%3A:
> In case of image or chart dependencies updates, those need to be bugfix only as well